src/x/ident/types.go

```diff
@@ -310,3 +311,30 @@ func (t Tags) Equal(other Tags) bool {
 	}
 	return true
 }
+
+// IDBatch is a batch of IDs that is consumed asynchronously.
+type IDBatch struct {
+	sync.WaitGroup
+	// IDs are the IDs for the batch.
+	IDs []ID
+}
+
+// IndexChecksumBlock represents a set of index checksums within a series block.
+type IndexChecksumBlock struct {
+	// Checksums is the list of index checksums.
+	Checksums []int64
+	// Marker is a batch marker, signifying the ID of the last element in the batch.
+	Marker []byte
```

Comment:
This will be sent over the wire as a raw `[]byte` (or rather this entire type will be compressed to a single byte buffer), so best to keep it as raw as possible here; potentially could cast it to `ident.BytesID` but then there may be accidental usage of its methods